core/src/consensus/tendermint/engine.rs

```diff
@@ -238,6 +238,11 @@ impl ConsensusEngine for Tendermint {
 
         stake::on_term_close(block.state_mut(), block_number, &inactive_validators)?;
 
+        match term_common_params.expect("TermCommonParams should exist").era() {
```

Comment:
Is there any reason this block didn't go inside the `stake::on_term_close` function?

Reply:
It is not 'stake' related operations.